search/search_integration_tests/processor_test.cpp

```diff
@@ -492,7 +492,6 @@ UNIT_CLASS_TEST(ProcessorTest, TestPostcodes)
   // Tests that postcode is added to the search index.
   {
     MwmContext context(m_engine.GetMwmHandleById(countryId));
-    TEST(context.IsAlive(), ());
```

Comment:
mpimenov <notifications@github.com> writes:

> In search/search_integration_tests/processor_test.cpp:
>
>> @@ -492,7 +492,6 @@ UNIT_CLASS_TEST(ProcessorTest, TestPostcodes)
>>    // Tests that postcode is added to the search index.
>>    {
>>      MwmContext context(m_engine.GetMwmHandleById(countryId));
>> -    TEST(context.IsAlive(), ());
>
> А что поменялось?

This CL removes MwmContext::IsAlive(). MwmContext::IsAlive() is broken
by design.

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>

-- 
Best regards, Yuri